app/lib/max_allowed_days_loader.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module MaxAllowedDaysLoader
+  module_function
+
+  def load_account_suspension
+    config[:account_suspension]
+  end
+
+  def load_account_inactivity
```

Comment:
I think we do not need those methods
We generally use `ActiveSupport::OrderedOptions`